src/functionality.js

```diff
@@ -0,0 +1,29 @@
+// Define a function to save the tasks in local storage
+export const saveTasks = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+// Define the functions for adding, deleting and editing tasks
+export const addTask = (description, tasks) => {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length,
```

Comment:
- [ ] Good job so far 👏 But I noticed that your last index number is not equal to the number of tasks because your index starts with **0** instead of **1**. According to the project requirements, **the property index is set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5).** Please fix it in order to make your project work as expected 👍 

![todoList](https://user-images.githubusercontent.com/6615394/230502377-e7fa2fc7-ddc2-4785-8014-e8c71febbed7.PNG)

Reply:
@codecaiine Done it. Now it looks like this: `index: tasks.length + 1`